media/themes/modern.msstyles/lang/en-US.rc

```diff
@@ -0,0 +1,39 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+STRINGTABLE
+{
+1000, "Light"
+1001, "Dark"
+}
+
+STRINGTABLE
+{
+2000, "Modern Light"
+2000, "Modern Dark"
+}
+
+STRINGTABLE
+{
+3000, "Regular Fonts"
+3001, "Large Fonts"
+3002, "Extra Large Fonts"
+}
+
+STRINGTABLE
+{
+4000, "Regular Fonts"
+4001, "Large Fonts"
+4002, "Extra Large Fonts"
+}
+
+STRINGTABLE
+{
+5000, "Modern"
+5001, "Visual Style"
+5002, "ReactOS Foundation"
+5003, "Pisarz, based on Luna Inspirat"
```

Comment:
`5003` means author. You could also add (just like the author from lautus did) indicate where you based it off, which would be lautus in this case.